src/main/java/core/basesyntax/service/impl/FileWriterServiceImpl.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.service.FileWriterService;
+import java.io.BufferedWriter;
+import java.io.FileWriter;
+import java.io.IOException;
+
+public class FileWriterServiceImpl implements FileWriterService {
+    private static final String FILE_NOT_WRITTEN_ERROR_MESSAGE = "Can't write data to the file: ";
+
+    @Override
+    public void write(String data, String toFileName, String path) {
```

Comment:
The method signature of 'write' should not include a separate 'path' parameter according to the checklist. It's better to use the 'toFileName' parameter as the relative path to the file.

Reply:
++